services/cli/src/bencher/sub/project/run/ci/github_actions.rs

```diff
@@ -187,11 +195,12 @@ impl GitHubActions {
                 self.ci_number.ok_or(GitHubError::NoWorkflowRunPRNumber)?
             },
             _ => {
-                cli_println_quietable!(
-                    log,
-                    "Not running as an expected GitHub Action event (`pull_request`, `pull_request_target`, or `workflow_run`). Skipping CI integration.\n{}",
+                cli_println!(
```

Comment:
This needs to still be `cli_println_quietable`. See: https://bencher.dev/docs/explanation/bencher-run/#--quiet